### PR TITLE
use accss_token and support drone ci

### PR DESCRIPTION
**Pull request by iflamed**
1. use access_token to access gitea resource;
2. add readme to show how to integration with Droneci.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,2 @@
...
-COPY . .
-ENV GO111MODULE=on
-RUN go build -o $GOPATH/bin/gitea-golangci-lint ./...
-
-# [Final image]
-FROM golang:alpine
-COPY --from=builder /go/bin/golangci-lint /go/bin/golangci-lint
-COPY --from=builder /go/bin/gitea-golangci-lint /go/bin/gitea-integration
+FROM alpine:latest
+COPY ./gitea-golangci-lint /bin/gitea
```
Comment (exepirit) [inline Dockerfile:2]: Is it better to build whole application inside container instead of copy prebuilt binary file.

**Review by exepirit — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (iflamed) [reply Dockerfile:2]: Yes, if we use multistage dockerfile will be much more convenient. But the dockerfile will be more complicated, it's not necessary. Mostly, we just build this docker image one time. It's not updated so frequently. So just create a simple docker file.

**Review by exepirit — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (exepirit) [reply Dockerfile:2]: I understood meaning of this change :)
But GitHub workflow uses build inside container, so multistage image build need.

**Review by exepirit — CHANGES_REQUESTED** [review]

**Review by exepirit — APPROVED** [review]

**Comment by exepirit** [comment]
I built docker image with your fixes and put it into DockerHub. Please use this version in your pipelines.
> docker pull exepir1t/gitea-golangci-lint:v0.2.0

**Comment by iflamed** [comment]
> I built docker image with your fixes and put it into DockerHub. Please use this version in your pipelines.
> 
> > docker pull exepir1t/gitea-golangci-lint:v0.2.0

Your docker image size is too big. I think you should remove the golangci-lint from the docker image. If you make it, the docker image size will be 10MB below. Also, I think you should not include the golangci-lint, so that we can use golangci-lint docker image separately. In this way, we can update golangci-lint docker image without update gitea-golangci-lint image.

**Comment by exepirit** [comment]
> Your docker image size is too big. I think you should remove the golangci-lint from the docker image.

In release v0.2.1 `golangci-lint` binary removed and container image size reduced to 14.5 MB.

**Comment by iflamed** [comment]
> > Your docker image size is too big. I think you should remove the golangci-lint from the docker image.
> 
> In release v0.2.1 `golangci-lint` binary removed and container image size reduced to 14.5 MB.

👍，Now all is going to be better.

**Pull request merged**